circuits/integration/mpc_circuits/match.rs

```diff
@@ -60,56 +59,65 @@ fn test_match_no_match(test_args: &IntegrationTestArgs) -> Result<(), String> {
         .map_err(|err| format!("Error allocating balance2 in the network: {:?}", err))?;
 
     // Build the test cases for different invalid match pairs
-    let mut test_cases: Vec<Vec<u64>> = vec![
+    let test_cases: Vec<(Order, u64)> = vec![
```

Comment:
still not super familiar w/ the difference between `mpc_circuits` and `zk_circuits` testing suites, but should we be adding test cases here that check for price protection violations?

Reply:
We run a raw MPC before the ZK-MPC that effectively serves to compute the witness for the multiprover ZKP. This test suite tests *only* the raw MPC, so it does not interact with price protection